payments-ui-core/res/values/strings.xml

```diff
@@ -96,4 +96,6 @@
   <string name="stripe_paymentsheet_payment_method_sepa_debit">SEPA Debit</string>
   <!-- Label of a button displayed below a payment method form. Tapping the button sets the payment method up for future use -->
   <string name="stripe_setup_button_label">Set up</string>
+  <!-- Title of a button with a 🔍 (magnifying glass) icon that starts a search when tapped -->
+  <string name="stripe_address_search_content_description">Search</string>
```

Comment:
Lint doesn't like the missing translations. How do I fix that?

Reply:
It’s a valid complaint. We should request translation in Lokalise and then pull in the translations via `localize.sh`.